share/emacs/modulefiles-mode.el

```diff
@@ -0,0 +1,51 @@
+;;; modulefiles-mode.el --- Major mode for Modules modulefiles editing -*- lexical-binding: t; -*-
+
+;; Author: Laurent Besson <laurent.42.besson@gmail.com>
+;; Maintainer: Laurent Besson <laurent.42.besson@gmail.com>
+;; Created: June 12, 2025
+;; Version: 1.0
+;; Package-Requires: ((emacs "24")
```

Comment:
do you confirm that emacs 24 is the oldest version that could run such mode file?